### PR TITLE
Fix intersection analysis

### LINKED ISSUE #8112: Intersection analysis fixes
- [x] Fix target list
- [x] Add `count` aggregation operation
- [x] Fix form labels (see design)
- [x] Remove old Point-in-polygon

### PR DESCRIPTION
**Pull request by javierarce**
- [x] Fix target list to accept sources from the user account
- [x] Add count aggregation operation
- [x] Fix form labels to reflect design
- [x] Remove point-in-polygon analysis

Closes #8112

### CODE REVIEW

**Comment by javierarce** [comment]
CR: @xavijam

**Comment by saleiva** [comment]
CR @xavijam

File: lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/intersection-form-model.js [hunk 1]
```diff
@@ -85,7 +90,7 @@ module.exports = BaseAnalysisFormModel.extend({
       aggregate_function: {
         type: 'Select',
         title: _t('editor.layers.analysis-form.operation'),
-        options: ['sum', 'avg', 'min', 'max'],
+        options: ['count', 'sum', 'avg', 'min', 'max'],
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/intersection-form-model.js]: could we internationalize them? I feel @MariaCheca already included them.

File: lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/intersection-form-model.js [hunk 2]
```diff
@@ -102,5 +107,45 @@ module.exports = BaseAnalysisFormModel.extend({
...
+
+  _getSourceOptionsForSource: function (sourceAttrName, requiredSimpleGeometryType) {
+    var currentSource = this.get(sourceAttrName);
+
+    if (this._isPrimarySource(sourceAttrName)) {
+      return [currentSource];
+    } else if (this._isFetchingOptions()) {
+      return [{
+        val: currentSource,
+        label: 'loading…'
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/intersection-form-model.js]: Same, let's create a "global" international string :).

**Comment by xavijam** [comment]
:clap: :clap: Just the i18n things :)

**Pull request merged**